capsules/extra/src/nmea_i2c.rs

```diff
@@ -0,0 +1,183 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2022.
+
+//! A capsule for I2C NMEA. This is used to read NMEA sentences from a GPS/GNSS
+//! device connected via I2C.
+//!
+//! This has been tested against a MediaTek device. It is expected to work with
+//! and GNSS device that reports NMEA strings when reading from an I2C address.
```

Comment:
Let's be more specific if possible. Which MediaTek device? Please update the comment.